### PR TITLE
upgradehandler for v1.2.10

### PR DESCRIPTION
**Pull request by ajansari95**
## 1. Summary
Fixes # (issue)
- closes `stargaze.delegate` channel for RR fix logic to be invoked

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

**Review by joe-bowman — COMMENTED** [review]
one minor query, otherwise lgtm

File: app/upgrades.go [hunk 1]
```diff
@@ -148,3 +152,30 @@ func v010209UpgradeHandler(app *Quicksilver) upgradetypes.UpgradeHandler {
...
+			panic(err)
+		}
+
+		ctx.EventManager().EmitEvents(sdk.Events{
+			sdk.NewEvent(
+				sdk.EventTypeMessage,
+				sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
+			),
+			sdk.NewEvent(
+				types.EventTypeReopenICA,
```
[the remark this answers is not in the document]
Comment (ajansari95) [reply app/upgrades.go:173]: https://github.com/ingenuity-build/quicksilver/blob/develop/x/interchainstaking/keeper/msg_server.go#L226 came from here

Comment (ajansari95) [inline app/upgrades.go:173]:
```suggestion
				types.EventTypeCloseICA,
```

Comment (ajansari95) [reply app/upgrades.go]: [668eaf7](https://github.com/ingenuity-build/quicksilver/pull/430/commits/668eaf7acea8885f8aa1996ff66c26092c8548fe)

**Review by aljo242 — APPROVED** [review]

**Review by joe-bowman — APPROVED** [review]

**Pull request merged**